### PR TITLE
Add support for custom translations

### PR DESCRIPTION
**Pull request by fjvallarino**
Templates Library subscription message is not exactly the same as the standard (the component does not show the name of the product), so the component had to be updated to take alternative translations. The current approach requires all the translations to be provided; a better way would have been only providing the modified ones, but it required a more complex logic on the component. We can improve it if deemed necessary.

This PR also fixes some issues caused by newer versions of widget-tester and Chrome dependencies.

@Rise-Vision/apps please review

### CODE REVIEW

File: src/subscription-status-template.html [hunk 1]
```diff
@@ -5,44 +5,44 @@ <h3 ng-disable-right-click>
```
Comment (ezequielc) [inline src/subscription-status-template.html]: @fjvallarino To confirm, it is not necessary to use {{messagesPrefix}} on line 3 above in ng-bind-html? Thanks

Comment (fjvallarino) [inline src/subscription-status-template.html]: @ezequielc good catch! I'll fix it now

Comment (fjvallarino) [inline src/subscription-status-template.html]: @ezequielc updated

**Comment by ezequielc** [comment]
@fjvallarino 👍

**Pull request merged**